modules/rhel-images-aws.adoc

```diff
@@ -41,9 +41,11 @@ This account ID is required to display AMI IDs for images that are provided by R
 ====
 When creating a {op-system-base} compute machine for AWS, ensure that the AMI is {op-system-base} 7.9.
 ====
-+
+
```

Comment:
This is outside the scope of this PR, but in looking at this file, I can see that the numbering for the callouts after the above code block is restarting. Would you mind looking into fixing that (either in this PR or in a future PR)?